### PR TITLE
check if inputs are empty to display a confirmModal or not

### PR DESCRIPTION
**Pull request by MilanBarande**
I just transformed voteProposalForm in a class based component to be able to use the local state in order to keep track of the inputs' content. If the fields are empty we don't ask to confirm deletion with a modal

### CODE REVIEW

**Comment by vincentfretin** [comment]
I'll leave it to @cedricmessiant to review this. :)

**Comment by cedricmessiant** [comment]
Why do you need a local state to keep track of the inputs ? you have the props for this, right ?

File: assembl/static2/js/app/components/administration/voteSession/voteProposalForm.jsx [hunk 1]
```diff
@@ -116,6 +116,10 @@ const DumbVoteProposalForm = ({
     displayCustomModal(content, true, 'gauge-settings-modal');
   };
 
+  const isTitleEmpty = title === '' || title === null;
+  const isDescriptionEmpty = description === null || description.blocks.every(block => block.text === '');
```
Comment (cedricmessiant) [inline assembl/static2/js/app/components/administration/voteSession/voteProposalForm.jsx]: title & description may not be undefined ?

Comment (cedricmessiant) [reply assembl/static2/js/app/components/administration/voteSession/voteProposalForm.jsx]: you should probably use `rawContentStateIsEmpty` (in `utils/draftjs.js`) that will work if there is an atomic block in your raw content state

Comment (MilanBarande) [reply assembl/static2/js/app/components/administration/voteSession/voteProposalForm.jsx]: for title & description they are set to null before anything is typed in, do you see a case where they would turn to undefined ? 
I'll check rawContentStateIsEmpty ;)

Comment (cedricmessiant) [reply assembl/static2/js/app/components/administration/voteSession/voteProposalForm.jsx]: no, it should be ok

**Pull request merged**